src/sentry/static/sentry/app/index.js

```diff
@@ -154,6 +154,8 @@ export default {
     Pagination: require('app/components/pagination').default,
     PluginConfig: require('app/components/pluginConfig').default,
     ProjectSelector: require('app/components/projectHeader/projectSelector').default,
+    ReleaseLanding: require('app/views/projectReleases/releaseLanding').default,
```

Comment:
Is there a related getsentry PR?

Reply:
Yes! https://github.com/getsentry/getsentry/pull/2282